.gitignore

```diff
@@ -0,0 +1,19 @@
+# Binary output dir
+bin/
+build/
+e2e/bin/
+
+# binary at the top
+/multi-networkpolicy-iptables
```

Comment:
please update to the right binary file

Reply:
It seems unneeded. Makefile will output to `build` directory.